detox/index.d.ts

```diff
@@ -264,109 +323,146 @@ declare global {
 
         type DetoxKnownDeviceType = DetoxBuiltInDeviceConfig['type'];
 
-        type DetoxConfigurationOverrides = {
-            artifacts?: false | DetoxArtifactsConfig;
-            behavior?: DetoxBehaviorConfig;
-            session?: DetoxSessionConfig;
-        };
+        type DetoxConfiguration = DetoxConfigurationCommon & (
```

Comment:
Yeah, damn, how this irritates me as well, oh, Asaf. I hated this silly collision since day 1 when I came to the Detox project.

I've been sitting on a fence for a long time already regarding introducing new terms to describe it better, like Detox.GlobalConfig and Detox.LocalConfig. And yet I'm not sure I'm ready to introduce more breaking changes: see our Migration Guide for v20, it is so long already. :(

Reply:
I see, you're right.. :/ at least I'm not alone in this opinion

Reply:
Added this as an issue: https://github.com/wix/Detox/issues/3667